test/steps/data.go

```diff
@@ -170,14 +170,15 @@ func logScenarioDuration(data *Data) {
 
 func handleScenarioResult(data *Data, scenario *godog.Scenario, err error) {
 	newLogFolderName := fmt.Sprintf("%s - %s", strings.ReplaceAll(scenario.GetName(), "/", "_"), data.Namespace)
+	var parentLogFolder string
 	if err != nil {
 		framework.GetLogger(data.Namespace).Error(err, "Error in scenario", "scenarioName", scenario.GetName())
-
-		newLogFolderName = "error - " + newLogFolderName
+		parentLogFolder = "errors"
 	} else {
+		parentLogFolder = "success"
```

Comment:
Maybe both singular or both plural would be better? Now it is errors and success.

Reply:
update to `error`